pediatric_rootlets/pediatric_rootlets.sh

```diff
@@ -0,0 +1,182 @@
+}
+
+# SCRIPT STARTS HERE
+# ==============================================================================
+# Display useful info for the log, such as SCT version, RAM and CPU cores available
+sct_check_dependencies -short
+
+# Go to folder where data will be copied and processed
+cd $PATH_DATA_PROCESSED
+
+# Copy source images
+# Note: we copy only T2w to save space
+rsync -Ravzh ${PATH_DATA}/./${SUBJECT}/anat/${SUBJECT//[\/]/_}_*T2w.* .
+
+# Go to anat folder where all structural data are located
+cd ${SUBJECT}/anat
+
+# Define the names of the T2w files
+file_t2_composed=${SUBJECT}_rec-composed_T2w
+file_t2_top=${SUBJECT}_acq-top_run-1_T2w
+
+# Check if file_t2_composed exists, if not, use file_t2_top as file_t2
+if [ -f ${file_t2_composed}.nii.gz ]; then
+    file_t2=${file_t2_composed}
+
+# Use `acq-top` if the composed image does not exist. This is the case, for example, for `sub-125`.
+else
+    file_t2=${file_t2_top}
+    echo "Composed T2w file not found. Proceeding only with Top T2w file."
+fi
+
+ # Segment spinal cord (only if it does not exist)
+segment_if_does_not_exist ${file_t2}.nii.gz
+
+# Run sct_label_vertebrae for vertebral levels estimation
+label_if_does_not_exist ${file_t2}.nii.gz
+
+# Project the intervertebral disc labels to the spinal cord centerline
+sct_label_utils -i ${FILESEG}.nii.gz -o ${FILELABEL}_centerline.nii.gz -project-centerline ${FILELABEL}.nii.gz -qc ${PATH_QC} -qc-subject ${SUBJECT}
+
+# Detect PMJ (only if it does not exist)
+detect_pmj_if_does_not_exist ${file_t2}.nii.gz
+
+# Crop composed images and labels to the size of the top image (if the composed image exists)
+if [ -f ${file_t2_composed}.nii.gz ]; then
+  $SCT_DIR/python/envs/venv_sct/bin/python ~/code/model-spinal-rootlets/pediatric_rootlets/crop_composed_images.py -i_composed ${file_t2_composed}.nii.gz -s ${FILESEG}.nii.gz -d ${FILELABEL}_centerline.nii.gz -pmj ${FILEPMJ}.nii.gz -x 11
+
+  # Run model-spinal-rootlets_ventral_D106_r20240523 for dorsal and ventral rootlets segmentation
+  # NOTE: as the model for both ventral and dorsal rootlets is not part of SCT yet, we run directly the nnUNet model using
+  # the wrapper script run_inference_single_subject.py from the model-spinal-rootlets repository
+  # https://github.com/ivadomed/model-spinal-rootlets/blob/main/packaging_ventral_rootlets/run_inference_single_subject.py
+  # NOTE: we use SCT python because it has nnUNet installed
+  # NOTE: the command below expects that you downloaded the model (https://github.com/ivadomed/model-spinal-rootlets/releases/tag/r20240523) and saved it to:  ~/models/model-spinal-rootlets_ventral_D106_r20240523
+  $SCT_DIR/python/envs/venv_sct/bin/python ~/code/model-spinal-rootlets/packaging_ventral_rootlets/run_inference_single_subject.py -i ${file_t2}_crop.nii.gz -o ${file_t2}_crop_label-rootlets_dseg.nii.gz -path-model ~/models/model-spinal-rootlets_ventral_D106_r20240523/model-spinal-rootlets_ventral_D106_r20240523 -fold all
```

Comment:
As we are about to push corrected rootlets under `derivatives`, it would be great to wrap this command under a function checking whether manually corrected labels already exist, something similar to `segment_if_does_not_exist()`